client/iface/wgproxy/factory_usp.go

```diff
@@ -10,6 +12,7 @@ type USPFactory struct {
 }
 
 func NewUSPFactory(iceBind *bind.ICEBind) *USPFactory {
+	log.Infof("WireGuard Proxy Factory will produce bind proxy")
```

Comment:
Should these new logs be `Debug` level?

Reply:
No, this log line is really important to know in any case. And it will appear only once time. We support 3 different implementation and good to know what is the active.